examples/dt/compute/amd-sev/edpm/nova_amd.yaml

```diff
@@ -0,0 +1,13 @@
+#
+#test amd sev 
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: amd-sev
+data:
+  01-amd-sev.conf: |
+    [libvirt]
+    virt_type=kvm
+    hw_machine_type=x86_64=q35
```

Comment:
You also need to include the definition for OpenstackDataplaneService, e.g.:  L30-43 https://github.com/openstack-k8s-operators/architecture/pull/62/files#diff-290c7c2d0a4eafe39f68ff8bdd28366368d0afd4d0fa3686944980e588f5b434

Reply:
added

Reply:
so hw_machine_type=x86_64=q35 is already done here

https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf#L179

but we do not define virt_type=kvm by default.

we probably should add that to 
https://github.com/openstack-k8s-operators/nova-operator/blob/main/templates/nova.conf give we don't officially support qemu in our product

and instead enable qemu in the ci jobs where its needed